pages/_app.tsx

```diff
@@ -10,11 +11,22 @@ import { AnimateSharedLayout } from 'framer-motion'
 import { QueryClient, QueryClientProvider } from 'react-query'
 import { useA11yFocusRing } from '~lib/hooks'
 import { theme, globalStyles } from '~ui/theme'
+import supportedBrowsers from '~lib/supportedBrowsers'
 
 const queryClient = new QueryClient()
 
 const App: React.FC<AppProps> = ({ Component, pageProps }) => {
   useA11yFocusRing()
+
+  if (
+    typeof navigator != 'undefined' &&
+    !supportedBrowsers.test(navigator.userAgent)
+  ) {
+    alert(
+      'Ihr Browser wird nicht unterstützt. Die Seite wird eventuell nicht funktionieren.'
```

Comment:
We need to decide if this is the correct way and wording for us to show this warning.

Reply:
Looks good from the just inform the user but yes I'm not sure if we should so do something else. I'll think and research a bit about it.

Reply:
Kira wanted to investigate the exact wording here. Waiting for her input on this.

Reply:
Thats nice :) 

From the implementation side I have no more comments. One way or another lead to this for what I've checked.